packages/wdio-devtools-service/src/utils.js

```diff
@@ -10,6 +10,11 @@ const log = logger('@wdio/devtools-service:utils')
 const RE_DEVTOOLS_DEBUGGING_PORT_SWITCH = /--remote-debugging-port=(\d*)/
 const RE_USER_DATA_DIR_SWITCH = /--user-data-dir=([^-]*)/
 const VERSION_PROPS = ['browserVersion', 'browser_version', 'version']
+const SUPPORTED_BROWSERS_AND_MIN_VERSIONS = {
+    'chrome': 63,
+    'chromium' : 63,
+    'google chrome': 63
```

Comment:
I have also seen this capability being used:
```suggestion
    'googlechrome': 63,
    'google chrome': 63,
```

Reply:
fixed